backend/endpoints/rooms.py

```diff
@@ -75,6 +78,75 @@ async def create_room(
         detail="unable to create room",
     )
 
+@router.patch("/org/{org_id}/rooms/{room_id}/members/{member_id}",
+    response_model=ResponseModel, 
+    status_code=status.HTTP_200_OK,
+    responses={
+        200: {"detail": {"room": "room"}},
```

Comment:
Successful response should no longer be here. 
It's already indicated `response_model`

Reply:
@zxenonx what do you mean by successful response should no longer be here,

Reply:
@kinglighthill, this line 200: {"detail": {"room": "room"}} should not appear in the `responses` dict